neurobooth_terra/dataflow.py

```diff
@@ -315,18 +315,16 @@ def delete_files(db_table, target_dir, suitable_dest_dir1, suitable_dest_dir2,
         is_deleted=True.
     target_dir : str
         The source directory path from which to delete files.
-    suitable_dest_dir1 : str
-        The destination directory path where the files should have been copied.
-    suitable_dest_dir2 : str
-        The destination directory path where the files should have been copied.
+    suitable_dest_dir : list
```

Comment:
If this is a list the name should probably be plural for readability.

Reply:
Thanks, done!